h-youjin/LV.1/2016년.js

```diff
@@ -0,0 +1,5 @@
+function solution(a, b) {
+  const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
+  let day = new Date(2016, a - 1, b);
+  return week[day.getDay()];
+}
```

Comment:
```suggestion
function solution(a, b) {
  const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
  let day = new Date(2016, a - 1, b);
  return week[day.getDay()];
}
```
제 풀이와 비슷한 코드네요! 유진님의 풀이 잘보고갑니다!